src/Classes/Controllers/GetEventsController.php

```diff
@@ -42,6 +42,24 @@ public function __invoke(Request $request, Response $response, array $args)
         if (!empty($eventSearchInput)) {
             if (strlen($eventSearchInput) < 256) {
             // run the search event function
```

Comment:
Ignore this file's changes - it is part of the next task branch and shouldn't affect this branch which is only to create the searchEvents method in the EventModel class